---
title: "Systems genetics of DNA damage tolerance"
subtitle: "Cisplatin, RAD5 & CRISPR-mediated nonsense"
author: "Eric Bryant"
date: "2020-01-09"
slug: "phd-dissertation"
bibliography: 
  - ../../static/references/library.bib
  - ../../static/references/thesis.bib
link-citations: yes
output: blogdown::html_page
categories:
  - Dissertation
tags:
  - Dissertation
  - Thesis
  - "My publications"
---



<p>The following post includes the <a href="#abstract">Abstract</a>, <a href="#preface">Preface</a> and <a href="#discussion">Discussion</a> from my
PhD thesis dissertation <span class="citation">(Bryant <a href="#ref-2019-01-Bryant">2019</a>)</span>.
I have included them here as a convenient place to share my thoughts and
perspective on my dissertation work.</p>
<ul>
<li><strong><a href="#abstract">Abstract</a></strong> – a high level summary of my thesis.</li>
<li><strong><a href="#preface">Preface</a></strong> – an historical and personal introduction to each
chapter of my thesis.</li>
<li><strong><a href="#discussion">Discussion</a></strong> – informal lessons and insights gained during my
dissertation work.</li>
<li><strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a></strong> – <em>Epistasis and cisplatin tolerance</em> –
currently available in the published thesis <span class="citation">(Bryant <a href="#ref-2019-01-Bryant">2019</a>)</span>.</li>
<li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a></strong> – <em>Genomic repercussions of RAD5 overexpression</em> –
published and open-access <span class="citation">(Bryant et al. <a href="#ref-2019-09-Bryant-Reid">2019</a>)</span>.</li>
<li><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">Chapter 3</a></strong> – <em>CRISPR-mediated nonsense</em> –
published and open-access <span class="citation">(Billon et al. <a href="#ref-2017-09-Billon-Bryant-Ciccia">2017</a>)</span>.</li>
</ul>
<div id="abstract" class="section level1 unnumbered">
<h1>Abstract</h1>
<p>DNA sequence information is constantly threatened by damage.
In the clinic, intentional DNA damage is often used to treat cancer.
Cisplatin, a first-line chemotherapy used to treat millions of patients,
functions specifically by generating physical links within DNA strands,
blocking DNA replication, and killing dividing cells.
To maintain genome integrity, organisms have evolved the capacity to repair,
respond, or otherwise resist change to the DNA sequence through a network of
genetically encoded DNA damage tolerance pathways.
In <a href="https://doi.org/10.7916/d8-k1d0-kb09">chapter 1</a>, I present advances in experimental design and
current progress for a systems genetics approach, using <em>Saccharomyces
cerevisiae</em>, to reveal relationships between cisplatin tolerance pathways.
Additionally, recent efforts to sequence thousands of cancer genomes have
revealed recurrent genetic changes that cause overexpression of specific
cisplatin tolerance genes.
In <a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">chapter 2</a>, I present a submitted manuscript that models
overexpression of an essential cisplatin tolerance gene.
This study uses a systems genetics approach to reveal the genetic pathways
that are essential for tolerating this perturbation, which ultimately led
to mechanistic insights for this gene.
Convenient genome engineering in <em>Saccharomyces</em> has made this organism an
ideal model to develop systems genetics concepts and approaches.
In <a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">chapter 3</a>, I present a published manuscript that demonstrates
a new approach to disrupting genes by making site-specific nonsense
mutations.
Importantly, this approach does not require cytotoxic double-strand DNA
breaks and is applicable to many model organisms for disrupting almost any
gene, which may advance systems genetics into new model organisms.
Systems genetics provides a framework for determining how DNA damage
tolerance pathways act together to maintain cellular fitness and genome
integrity.
Such insights may one day help clinicians predict which cancers will respond
to treatment, potentially sparing patients from unnecessary chemotherapy.</p>
</div>
<div id="preface" class="section level1 unnumbered">
<h1>Preface</h1>
<p>Deoxyribonucleic acid (DNA) holds the instructions for life on Earth.
Its code consists of only four letters –
Adenine (A), Guanine (G), Thymine (T) and Cytosine (C) –
the sequence of which determines ribonucleic acids (RNA) to be
transcribed, and proteins to be translated in living cells
<span class="citation">(Dahm <a href="#ref-2005-02-Dahm">2005</a>; Avery, Macleod, and McCarty <a href="#ref-1944-02-Avery-McCarty">1944</a>; Crick et al. <a href="#ref-1961-12-Crick-WattsTobin">1961</a>; Nirenberg et al. <a href="#ref-1966-00-Nirenberg-Anderson">1966</a>)</span>.
Transcribing DNA into RNA and translating RNA into protein forms the
foundation for the central dogma of molecular biology
<span class="citation">(Crick <a href="#ref-1970-08-Crick">1970</a>)</span>,
which emphasizes an important feature of DNA – its sequence can store the
information necessary to reproduce the chemistry of life.
This feature, along with fortuitous structural properties
<span class="citation">(Watson and Crick <a href="#ref-1953-04-Watson-Crick">1953</a>)</span>,
makes DNA a molecule that can not only reproduce the chemistry of life, but can
also be reliably copied and passed to the next generation.
However, to reliably and faithfully pass DNA to the next generation, cells
must overcome a constant barrage of DNA damage <span class="citation">(Lindahl <a href="#ref-1993-04-Lindahl">1993</a>)</span>.
Fortunately, the genomes of most cells contain instructions to build a large
number of proteins that can work together to repair broken DNA, or enable
DNA damage tolerance <span class="citation">(Cleaver <a href="#ref-2016-01-Cleaver">2016</a>)</span>.
This genetically encoded capacity for cells to maintain genome integrity and
viability in the face of damaged DNA is the focus of my thesis.
In particular, I am interested in understanding how cells resist treatment to
the DNA damaging agent, cisplatin – one of several platinum-based
chemotherapeutic drugs that have been used to treat cancer in millions of
patients
<span class="citation">(Galanski, Jakupec, and Keppler <a href="#ref-2005-00-Galanski-Keppler">2005</a>; Simon et al. <a href="#ref-2000-01-Simon-Friend">2000</a>; Wu et al. <a href="#ref-2004-06-Wu-Brown">2004</a>; Kelland <a href="#ref-2007-08-Kelland">2007</a>)</span>.</p>
<p>Decades of work by geneticists have revealed DNA damage tolerance to consist of
hundreds of genes that operate in numerous pathways that collectively enable
cells to resist DNA lesions caused by cisplatin treatment
<span class="citation">(reviewed by Ciccia and Elledge <a href="#ref-2010-10-Ciccia-Elledge">2010</a>; Heyer, Ehmsen, and Liu <a href="#ref-2010-00-Heyer">2010</a>; Rocha et al. <a href="#ref-2018-09-Rocha">2018</a>)</span>.
While the mechanisms for many of the major pathways responsible for cisplatin
tolerance are well established, how these genetic pathways relate to each
other as a system to maintain fitness and DNA integrity when responding to
cisplatin treatment remains poorly understood.
Moreover, many individual genes known to affect cellular fitness of cisplatin
treated cells have no known role in any of the major cisplatin tolerance
pathways.
And, while much is known about how loss of individual gene function can affect
cisplatin tolerance, recent efforts to genomically characterize thousands of
tumors has revealed recurrent DNA copy number amplification events that can
result in overexpression of specific genes <span class="citation">(Cancer Genome Atlas Research Network <a href="#ref-2008-10-TCGA">2008</a>)</span>.
The genomic repercussions of gene overexpression, and how this could alter
cisplatin tolerance, remains a largely unexplored area in genetics.</p>
<p>The work described here attempts to address some of these unexplored and poorly
understood aspects of DNA damage tolerance using systems genetics approaches.
<strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a></strong>, describes the experimental design and current
results for an ongoing collaborative effort to map all cisplatin specific DNA
damage tolerance pathway relationships by systematically measure fitness of
double-mutants to identify genetic interactions
(i.e. double-mutant combinations that cause an unexpected change in fitness).
<strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a></strong>, is a submitted manuscript that utilizes a genetic
interaction profile to reveal the genomic repercussions of <em>RAD5</em>
overexpression, one of the most important cisplatin tolerance genes, whose
human homolog is frequently overexpressed in specific cancers.
<strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">Chapter 3</a></strong>, is a published manuscript that describes a new
approach to efficiently disrupt the function of genes, which will open new
avenues for studying the systems genetics of DNA damage tolerance in a diverse
range of organisms.
An introduction to the scientific and thesis context for each chapter is
provided below in
<strong><a href="#intro-cispt">Chapter 1 introduction</a></strong>,
<strong><a href="#intro-rad5">Chapter 2 introduction</a></strong> &amp;
<strong><a href="#intro-istop">Chapter 3 introduction</a></strong>.
Ultimately, understanding the fundamental genetic determinants of cisplatin
tolerance may one day help clinicians predict which cancers will respond to
cisplatin treatment, thus informing treatment decisions, and sparing patients
from unnecessary chemotherapy <span class="citation">(O’Connor <a href="#ref-2015-11-Connor">2015</a>)</span>.</p>
<div id="intro-cispt" class="section level2">
<h2>Epistasis and cisplatin tolerance</h2>
<p>The field of genetics seeks to answer one simple question – how does DNA
sequence, combined with environmental exposure, result in the observable
features of an organism?
In other words, how does genotype, plus environment, produce
phenotype?
Historically, geneticists have approached the “genotype to phenotype” question
systematically using genetic screens.
In such screens, changes to an organism’s DNA sequence would be made randomly
using some kind of DNA damaging agent to produce mutants.
Mutants that gained the phenotype of interest would be cataloged, and the
tedious process of characterizing each mutant allele to identify the
causative DNA sequence change would commence.
The classic genetic screen began to lose prominence with the advent of
high-throughput sequencing technologies that allowed scientists to produce
the first complete sequence assemblies for the human genome and several
model organisms
<span class="citation">(Lander et al. <a href="#ref-2001-02-Lander-IHGSC">2001</a>)</span>.
Having sequenced a genome, scientists could computationally predict all
of an organism’s protein coding genes – no known phenotype was necessary.
Researchers could now perform their genetic screens in reverse.
Instead of asking, “given a phenotype, what are the alleles?” scientists
could now ask, “given the alleles, what are the phenotypes?”
Thus, the reverse genetic screen was born.</p>
<p>Geneticists studying the budding yeast, <em>Saccharomyces cerevisiae</em>, were early
adopters of the reverse genetic screen, which first required the construction
of a large collection of defined alleles.
Only a few short years after sequencing the yeast genome, every gene in yeast
had been individually deleted to produce the non-essential gene disruption
collection
<span class="citation">(Winzeler et al. <a href="#ref-1999-08-Winzeler-Davis">1999</a>; Giaever et al. <a href="#ref-2002-07-Giaever-Johnston">2002</a>)</span>.
Remarkably, of the 6,000 protein coding genes in the yeast genome,
approximately 5,000 were <em>not</em> essential for the organism to survive, which
left a big open question – are these genes important for anything?
The reverse genetic screen was the perfect tool for quickly answering this
question.
After exposing this strain collection to over 350 different chemicals, a
group of researchers reassuringly found that indeed these genes were
important, as 97% of genes showed a growth defect, or advantage, in at least
one treatment condition
<span class="citation">(Hillenmeyer et al. <a href="#ref-2008-04-Hillenmeyer-Giaever">2008</a>)</span>.
This reverse genetic screening strategy, known as chemogenomic profiling,
when used with a comprehensive collection of gene deletions, produces a list,
or “profile”, of genes that contribute to fitness when treated with a
particular chemical.
Importantly, information gained from chemogenomic profiling is a two-way
street; as we learn more about gene function, a chemogenomic profile can tell
us something about the biological impact of the chemical or treatment itself.
For example, early genetic screens in yeast from the 1960s and 1970s had
already identified genes that are essential to tolerate exposure to
ultraviolet, x-ray, and gamma radiation
<span class="citation">(Snow <a href="#ref-1967-09-Snow">1967</a>; Cox and Parry <a href="#ref-1968-00-Cox-Parry">1968</a>; Lemontt <a href="#ref-1971-05-Lemontt">1971</a>; Game and Mortimer <a href="#ref-1974-09-Game-Mortimer">1974</a>)</span>.
These genes were given names starting with “<em>RAD</em>” and given a number
(e.g. <em>RAD3</em>, <em>RAD6</em>, <em>RAD52</em>, etc.).
Later work had determined that many of the <em>RAD</em> genes produced proteins that
performed various DNA repair processes to combat DNA damage
<span class="citation">(Haynes and Kunz <a href="#ref-1981-00-Haynes-Kunz">1981</a>)</span>.
With this knowledge of <em>RAD</em> gene function in mind, when we generate a new
chemogenomic profile for an uncharacterized compound and find that its profile
includes many <em>RAD</em> genes, we can infer that this new compound likely causes
DNA damage.
Thus, the more knowledge we have of gene function, the more powerful
chemogenomic profiling becomes.</p>
<p>Cisplatin, being a critically important and clinically relevant drug, received
plenty of attention during the heyday of chemogenomic profiling
<span class="citation">(Birrell et al. <a href="#ref-2002-06-Birrell-Brown">2002</a>; Wu et al. <a href="#ref-2004-06-Wu-Brown">2004</a>; Lee et al. <a href="#ref-2005-08-Lee-Giaever">2005</a>; Huang et al. <a href="#ref-2005-07-Huang-Kowalski">2005</a>; Hastie et al. <a href="#ref-2006-07-Hastie-Cohen">2006</a>; Liao et al. <a href="#ref-2007-02-Liao-Panaretou">2007</a>)</span>.
However, while chemogenomic profiling has identified over 100 genes in yeast
that are important for cisplatin tolerance, this list of genes does not tell
us the biological process, or pathway, in which each gene participates.
Moreover, a chemogenomic profile alone cannot tell us how the collection of
pathways that affect fitness in response to cisplatin relate to each other as
a complete system.
For this, we leverage the power of double-mutants to identify which genes have
shared contributions, or compensatory contributions to cisplatin sensitivity.
This strategy was elegantly used by
<span class="citation">Brendel and Haynes (<a href="#ref-1973-09-Brendel-Haynes">1973</a>)</span>
to group genes known at the time to be involved in DNA repair into three major
pathways, later defined as the <em>RAD3</em>, <em>RAD6</em> and <em>RAD52</em> epistasis groups.
An epistasis group, in this context, refers to a set of mutations that share a
contribution to the phenotype of DNA damage tolerance and can thus be grouped
into shared damage tolerance pathways.
Epistasis itself is a class of genetic interaction, a relationship between two
genetic perturbations, that geneticists use to determine pathway membership.
In the context of fitness, epistasis is most often observed when two
deleterious genetic perturbations result in better fitness than expected when
the two perturbations are combined.
Thus, for clarity, I often describe such epistasis as a positive fitness
interaction.
In addition to identifying positive fitness interactions, double-mutant
analysis can also identify negative fitness interactions – pairs of mutations
that cause worse than expected fitness and indicate that each gene contributes
to fitness in parallel compensatory pathways.</p>
<p>The major limitation to double-mutant analysis is time and energy.
For example, in yeast, with a collection of 5,000 non-essential gene
deletions, a researcher would have to perform approximately 12.5 million
crosses to identify all possible fitness interactions among these gene
deletions for a single treatment condition.
Remarkably, after developing the Synthetic Genetic Array (SGA) method,
researchers managed to use cleverness and robotics to produce a genetic
interaction profile for nearly every gene deletion in budding yeast
<span class="citation">(Tong et al. <a href="#ref-2001-12-Tong-Boone">2001</a>; Tong and Boone <a href="#ref-2006-00-Tong-Boone">2006</a>; Pan et al. <a href="#ref-2006-03-Pan-Boeke">2006</a>; Costanzo et al. <a href="#ref-2010-01-Costanzo-Boone">2010</a>; Baryshnikova et al. <a href="#ref-2010-12-Baryshnikova-Myers">2010</a>)</span>.
The combined interaction profiles generate an interaction network that
represents the genetic landscape of <em>Saccharomyces cerevisiae</em>
<span class="citation">(Usaj et al. <a href="#ref-2017-00-Usaj-Boone">2017</a>; Costanzo et al. <a href="#ref-2016-00-Costanzo-Boone">2016</a>)</span>.</p>
<p>Unfortunately, this genetic landscape only reveals genetic relationships for
pathways that contribute to fitness in standard laboratory conditions.
To produce this network for the same drug treatment conditions used by
<span class="citation">Hillenmeyer et al. (<a href="#ref-2008-04-Hillenmeyer-Giaever">2008</a>)</span> to reveal a phenotype for nearly every single gene
deletion, one would have to phenotype all 12.5 million crosses 350 times,
generating over 4 billion measurements.
Even worse, gene deletions are only one type of genetic perturbation.
To generate an interaction landscape for every single nucleotide change in
yeast, and for 350 treatment conditions, one would need to generate more than
a quadrillion fitness measurements (&gt;10<sup>15</sup>).
Needless to say, a more focused approach for characterizing pathway
relationships for genes involved in cisplatin tolerance is warranted.
The strategy here is simple – focus on a set of genes that share a phenotype
as single mutants and only generate crosses among this small set of genes.
This strategy, when combined with SGA, is known as epistatic mini-array
profiling, or E-MAP
<span class="citation">(Schuldiner et al. <a href="#ref-2005-11-Schuldiner-Krogan">2005</a>, <a href="#ref-2006-12-Schuldiner-Krogan">2006</a>; Collins et al. <a href="#ref-2006-00-Collins-Weissman">2006</a>, <a href="#ref-2007-04-Collins-Krogan">2007</a>; Fiedler et al. <a href="#ref-2009-03-Fiedler-Krogan">2009</a>; Braberg et al. <a href="#ref-2013-08-Braberg-Krogan">2013</a>)</span>.
And, when combined with a treatment it is referred to as a differential E-MAP,
or dE-MAP
<span class="citation">(St. Onge et al. <a href="#ref-2007-02-Onge-Giaever">2007</a>; Bandyopadhyay et al. <a href="#ref-2010-12-Bandyopadhyay-Ideker">2010</a>; Gu’enol’e et al. <a href="#ref-2013-01-Gunol-vanAttikum">2013</a>; Srivas et al. <a href="#ref-2013-12-Srivas-Ideker">2013</a>; Hustedt et al. <a href="#ref-2015-01-Hustedt-Gasser">2015</a>)</span></p>
<p><strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a></strong> presents an advancement in the experimental design
for a cisplatin dE-MAP.
Currently published dE-MAPs typically use only one, or two, fixed
concentrations of drug treatment.
However, detectable sensitivity for cisplatin tolerance mutants ranges across
nearly two orders of magnitude of cisplatin concentration.
For example, the most sensitive mutants, <em>rad5∆</em>, <em>rad6∆</em> and <em>rad18∆</em>, have a
measurable phenotype at 2 µM cisplatin, and are completely dead at 25 µM.
Whereas, <em>psy4∆</em>, a gene named for cisplatin sensitivity, has a barely
detectable phenotype at 80 µM.
To measure a genetic interaction, ideally the experiment uses a treatment
condition where at least one of the two mutations has a phenotype, and neither
mutation causes death.
Therefore, to optimize detection of genetic interactions, we use a common
“low” concentration where most strains will remain viable, and an optimized
“high” concentration that produces a phenotype of the query mutation – the
gene deletion being crossed to the mini-array of cisplatin tolerance genes.
While this project remains ongoing, we have successfully used this strategy to
produce a high-resolution set of genetic interactions covering 70% of the
cisplatin tolerance network.</p>
</div>
<div id="intro-rad5" class="section level2">
<h2>Genomic repercussions of <em>RAD5</em> overexpression</h2>
<p>When I began my PhD, I spent a few months meeting with
professors looking for scientific inspiration.
Having a background in <em>de novo</em> gene synthesis, I was naturally drawn to the
topic of DNA recombination and repair, so I set up a meeting with Professor
Rodney Rothstein – a leader in this field.
During our meeting, Rodney described a project that had nothing to do with DNA
repair.
Instead, he described an ongoing project in his lab that aimed to identify
genetic interactions that could be used to treat cancer.
Fundamentally, the project was an extension of the genetic screening for
cancer drug targets approach described by
<span class="citation">Hartwell et al. (<a href="#ref-1997-11-Hartwell-Friend">1997</a>)</span>.
The described approach attempts to address a major problem in the current
strategy for treating cancers, which is that typical chemotherapy and
radiation treatment programs have many dangerous side-effects because they do
not specifically kill cancer cells, rather they preferentially kill all
dividing cells.
The logic of the genetic approach described by Hartwell, begins with the
observation that there are numerous recurring genetic alterations that are
observed in cancer, which represent an an attack surface for selectively
killing cancer cells.
Geneticists had long observed a phenomenon they called synthetic lethality,
wherein combining two otherwise harmless mutations results in cell death
(i.e. an extreme example of a negative fitness genetic interaction).
So the theory goes, if a cancer has a mutation and we know the synthetic
lethal interactions of that mutation, then we can selectively target the
cancer cell by inhibiting a synthetic lethal interactor with a drug.
This drug would selectively kill only cancer cells harboring the interacting
mutation, while leaving normal cells unharmed.
Remarkably, this approach has received some traction with the development of
clinically approved PARP inhibitors, which exploit a synthetic lethal genetic
interaction between <em>PARP</em> and commonly observed mutations in the breast
cancer susceptibility genes, <em>BRCA1</em> and <em>BRCA2</em>
<span class="citation">(Ashworth and Lord <a href="#ref-2018-09-Ashworth-Lord">2018</a>; Lord, Tutt, and Ashworth <a href="#ref-2015-00-Lord-Ashworth">2015</a>)</span>.
Needless to say, I found the synthetic lethality approach to targeted cancer
therapy to be incredibly interesting, so I asked to join the Rothstein lab.</p>
<p>The first major challenge to using synthetic lethality for treating cancer is
to identify common genetic alterations in cancer.
Then, by pairing these common genetic alterations with a collection of mutant
strains one can identify synthetic lethal interactions.
When I started in the Rothstein lab, the project I joined was attempting to
leverage yeast genetics to identify synthetic lethal interactions for genes
that are commonly overexpressed in PTEN deficient cancer.
After performing a number of these genetic screens, it became clear that the
data we were using to identify these overexpressed genes could not distinguish
between genes that are overexpressed in individual cells, and genes that are
just overrepresented in the population of cells used to measure expression
(e.g. S-phase regulated genes will be overrepresented in a sample that has
more dividing cells).
Sadly, this issue undermined the initial motivation for performing my first
synthetic lethal screens.
Undeterred, I looked to other sources to find data for frequent genetic
alterations in cancer.
Around this time, The Cancer Genome Atlas consortium <span class="citation">(Cancer Genome Atlas Research Network <a href="#ref-2008-10-TCGA">2008</a>)</span> had
released data for thousands of genomically characterized tumors.
Importantly, these data included not only gene expression information, but
also copy number information.
After diving into the data, it became clear that each cancer cohort had
distinct patterns of recurrent focal and chromosomal copy number
amplifications resulting in overexpression of specific genes.
Importantly, because gene copy number amplification is a physical change that
occurs within individual cells, the change in expression could be attributed
to an actual increase in transcript levels within cells rather than just a
population level change.</p>
<p>In <strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a></strong>, I present a submitted manuscript that was
motivated by the observation that the <em>RAD5</em> human homolog, <em>HLTF</em>, incurs
frequent copy number amplification and corresponding overexpression in several
types of human cancer (typically squamous cell carcinoma).
<em>RAD5</em> is a member of the most essential cisplatin tolerance pathway, and,
when overexpressed, we found that Rad5 bypasses canonical regulatory signals
to cause cisplatin sensitivity and genome instability by driving hyperactive
recombination at replication forks.</p>
</div>
<div id="intro-istop" class="section level2">
<h2>CRISPR-mediated nonsense</h2>
<p>Yeast has been an ideal model organism for studying genetic interactions at
scale to identify genetic determinants of phenotype, and to reveal pathway
relationships between genes.
One reason yeast has achieved such traction as a genetic model arises from the
ease with which scientists can modify the yeast genome in the lab.
The ability to make targeted changes in an organism’s genome emerged from
pioneering work in the field of genetic recombination, which led to the
development of one-step gene disruption – a fast and efficient approach to
knockout a gene
<span class="citation">(Rothstein <a href="#ref-1983-01-Rothstein">1983</a>; Baudin et al. <a href="#ref-1993-07-Baudin-Cullin">1993</a>; Wach et al. <a href="#ref-1994-12-Wach-Philippsen">1994</a>)</span>.
Eventually, with the release of the <em>S. cerevisiae</em> genome
<span class="citation">(Goffeau et al. <a href="#ref-1996-10-Goffeau-Oliver">1996</a>; Goffeau and Vassarotti <a href="#ref-1991-01-Goffeau-Vassarotti">1991</a>)</span> –
the first complete DNA sequence assembly of any eukaryotic organism –
work began to construct a comprehensive collection of yeast knockout mutants
representing nearly all of the roughly 5,000 non-essential genes found in
budding yeast
<span class="citation">(Winzeler et al. <a href="#ref-1999-08-Winzeler-Davis">1999</a>; Giaever et al. <a href="#ref-2002-07-Giaever-Johnston">2002</a>)</span>.
This collection paved the way for the development of reverse genetic
screening approaches, which utilize a large set of defined mutants to
identify those that give rise to a particular phenotypic feature, such as
cisplatin sensitivity (see <strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a></strong>), or a genetic interaction
profile (see <strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a></strong>).
In principle, one-step gene disruption should be feasible in any recombination
proficient organism, which includes most of the tree of life.
In fact, adaptation of the one-step gene disruption approach in mice was the
basis of the 2007 Nobel Prize in Physiology or Medicine
<span class="citation">(Vogel <a href="#ref-2007-10-Vogel">2007</a>; Koller et al. <a href="#ref-1990-06-Koller-Smithies">1990</a>; Zijlstra et al. <a href="#ref-1990-04-Zijlstra-Jaenisch">1990</a>, <a href="#ref-1989-11-Zijlstra-Jaenisch">1989</a>; Koller and Smithies <a href="#ref-1989-11-Koller-Smithies">1989</a>)</span>,
and the mouse genetics community has followed suit in sequencing the mouse
genome
<span class="citation">(Waterston et al. <a href="#ref-2002-12-Waterston">2002</a>)</span>,
and constructing their own large collections of knockout mice
<span class="citation">(Grimm <a href="#ref-2006-06-Grimm">2006</a>)</span>.
However, one-step gene disruption, and other recombination mediated genome
engineering methods, rely on rare DNA repair events, rendering these approaches
inefficient in most organisms – entire PhDs could be dedicated to
constructing a single knockout mouse.
Thus, at the beginning of my PhD in the fall of 2011, yeast remained a
dominant model in the field of systems genetics and was the focus of my
research on the genetic relationships between cisplatin tolerance pathways and
the genetic repercussions of <em>RAD5</em> overexpression.</p>
<p>However, in the spring of 2013, when I was working as a teaching assistant for
Professor Ron Prywes and Professor Songtao Jia’s course in molecular biology,
a fellow teaching assistant, Ling Ye, informed me of a new advancement in the
field of genome engineering that he was sure would change the future of
genetics research.
This new advancement in genome engineering technology that Ling described to
me is the now famous CRISPR-Cas9 system, which has indeed changed the face of
genetics and biomedical research
<span class="citation">(Jinek et al. <a href="#ref-2012-08-Jinek-Charpentier">2012</a>; Gasiunas et al. <a href="#ref-2012-09-Gasiunas-Siksnys">2012</a>; Deltcheva et al. <a href="#ref-2011-03-Deltcheva-Charpentier">2011</a>; Sapranauskas et al. <a href="#ref-2011-11-Sapranauskas-Siksnys">2011</a>; Garneau et al. <a href="#ref-2010-11-Garneau-Moineau">2010</a>; Cong et al. <a href="#ref-2013-02-Cong-Zhang">2013</a>; Mali et al. <a href="#ref-2013-02-Mali-Church">2013</a>)</span>.
The elegance of the CRISPR-Cas9 system for genome engineering is that it
consists of a single protein (Cas9) and a short programmable guide RNA that
targets Cas9 to a specific location in the genome where, upon recognition of
the target sequence, it will generate a DNA double-strand break (DSB).</p>
<p>Suddenly, that rare DNA repair event necessary for recombination mediated
genome engineering could be efficiently induced, opening up new avenues for
modifying the genomes of almost any organism.
Moreover, this simple system was amenable to simple modifications which allowed
scientists to localize other proteins to specific DNA sequences without causing
DNA breaks.
Using this approach,
<span class="citation">Komor et al. (<a href="#ref-2016-05-Komor-Liu">2016</a>)</span>
and others, modified the CRISPR system to localize an enzyme that would
directly convert cytosines (C) to thymines (T) in a small window of five
nucleotides near the guide targeted sequence without having to create a DSB –
an attractive feature for those interested in modifying cells that might
already be deficient in DNA repair.
When this system, known as CRISPR-mediated base editing, was first described, I
had recently begun a collaboration with Dr. Alberto Ciccia’s lab to help with
sequence analysis for a pilot run of a CRISPR-Cas9 based reverse genetic
screen.
Dr. Ciccia and a postdoctoral research associate in his lab, Dr. Pierre
Billon, realized that CRISPR-mediated base editing of C to T could be used to
disrupt gene function by converting four specific codons to premature stop
codons in protein coding genes.
What was unclear though, was whether this approach for knocking out gene
function would be generalizable given the limited number of targetable codons
and other technical restrictions dictated by the CRISPR system.</p>
<p><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">Chapter 3</a></strong> is a published manuscript I co-authored with
Dr. Billon and Dr. Ciccia addressing the scope and feasibility of CRISPR-mediated
base editing for disrupting gene function
<span class="citation">(Billon et al. <a href="#ref-2017-09-Billon-Bryant-Ciccia">2017</a>)</span>.
The title of the original publication – <em>CRISPR-mediated base editing enables
efficient disruption of eukaryotic genes through induction of STOP codons
(iSTOP)</em> – sums up the ultimate conclusion; CRISPR-mediated C to T base
editors allow DSB free induction of stop codons at 23 different codons per
gene on average
(i.e. 1 iSTOP targetable codon for every 26 codons in the human genome).
Thus, iSTOP provides a new tool alongside CRISPR-Cas9, and classic one-step
gene disruption, that no longer relies on DSBs and recombination to
efficiently knockout gene function.
iSTOP, and other gene editing tools, will enable scientists to look beyond the
humble baker’s yeast, to many other branches of life, to gain insight from
systems genetics into the genetic determinants of life.</p>
</div>
</div>
<div id="discussion" class="section level1">
<h1>Discussion</h1>
<p>Genome integrity is essential for organisms to faithfully pass genetic
information to future generations.
While genomic DNA is constantly subject to damage, organisms have evolved
numerous mechanisms to maintain sequence information and allow replication of
their genomes despite this damage.
Defects in DNA damage tolerance pathways, either inherited from birth, or
acquired during the life of an organism, can result in numerous diseases such
as Fanconi anemia, xeroderma pigmentosum, ataxia telangiectasia, Nijmegen
breakage, Cockayne, Werner &amp; Bloom syndromes <span class="citation">(Knoch et al. <a href="#ref-2012-08-Knock-Berneburg">2012</a>)</span>.
In addition to these diseases, defects in DNA damage tolerance often arise
during the development of cancer.
Perhaps the best known examples of this are mutations in the breast cancer
susceptibility genes, <em>BRCA1</em> and <em>BRCA2</em>, which are both thought to be
involved in homology directed repair
<span class="citation">(Roy, Chun, and Powell <a href="#ref-2011-12-Roy-Powell">2011</a>; Zhang <a href="#ref-2013-02-Zhang">2013</a>)</span>.
By leveraging knowledge of genetic interactions, new drugs have been developed
to selectively kill cancers harboring mutations in <em>BRCA</em> genes by exploiting
synthetic lethality between <em>PARP</em> and <em>BRCA</em>
<span class="citation">(Lord, Tutt, and Ashworth <a href="#ref-2015-00-Lord-Ashworth">2015</a>; Ashworth and Lord <a href="#ref-2018-09-Ashworth-Lord">2018</a>)</span>.
Long before it became clear that many cancers harbor mutations in DNA repair
pathways, oncologists have treated patients with DNA damage through radiation
and chemotherapy.
As DNA sequencing costs have dropped, ever more tumors are being sequenced
to identify common mutations and changes in gene expression, which has led to
an increased appreciation that the status of DNA damage tolerance pathways in
tumors can have a profound effect on a tumor’s response to our most common
cancer therapies
<span class="citation">(Alexandrov et al. <a href="#ref-2013-08-Alexandrov-Stratton">2013</a>; Dietlein, Thelen, and Reinhardt <a href="#ref-2014-08-Dietlein-Reinhardt">2014</a>)</span>.
As insight into the DNA damage tolerance network develops, strategies for
selectively killing repair deficient tumors with drug combinations designed
to target redundancies in the repair network are beginning to emerge
<span class="citation">(Cheng et al. <a href="#ref-2013-04-Cheng-Halmos">2013</a>; Nickoloff et al. <a href="#ref-2017-11-Nickoloff-Hromas">2017</a>)</span>.
While some tumors respond well to radiation and DNA damaging chemotherapies,
many tumors either do not respond, or acquire resistance to such treatments.
This phenomenon underscores the importance of determining how the genetic
landscape of a tumor impacts the overall damage tolerance network.
The emerging field of systems genetics
<span class="citation">(Baliga et al. <a href="#ref-2017-01-Baliga-Vonesch">2017</a>)</span>,
and specifically the study of genetic interaction networks,
offers an important perspective for researchers and clinicians trying to
understand how the DNA damage tolerance network enables tumor evolution while
maintaining cancer cell viability.
The structure of this network can enable precision medicine by providing
critical insights into pathway organization that can reveal tumor specific
genetic weaknesses <span class="citation">(Hartwell et al. <a href="#ref-1997-11-Hartwell-Friend">1997</a>)</span>.
Below, I will discuss some lessons and insights from my work on the systems
genetics of DNA damage tolerance – a stepping stone on the long path to
precision medicine in oncology.</p>
<div id="epistasis-undetected-nonexistent" class="section level2">
<h2>Epistasis: undetected ≠ nonexistent</h2>
<p>In <strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a></strong>, I presented progress toward constructing a
genetic landscape for the cisplatin tolerance network in <em>Saccharomyces</em>.
The design of this experiment advances the dE-MAP approach for detecting
treatment specific pathway membership, and overall pathway relationships
<span class="citation">(Bandyopadhyay et al. <a href="#ref-2010-12-Bandyopadhyay-Ideker">2010</a>; Ideker and Krogan <a href="#ref-2012-01-Ideker-Krogan">2012</a>)</span>.
Specifically, we took advantage of the null model for genetic interactions of
colony fitness to account for the fact that detection of genetic relationships
between two alleles critically depends on the phenotype of each allele
(<strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a>, Figure 2</strong>).
While this fact has been understood by geneticists for decades, it has some
unappreciated and important consequences for interpreting existing genetic
interaction networks.
Perhaps the most important consequence of requiring a phenotype to detect
epistasis is that an undetected interaction is <em>not</em> equivalent to no
interaction.
In fact, conspicuous absence of an expected synergistic interaction may be
weak evidence for epistasis in an as yet unknown treatment condition.
Two epistatic alleles will <em>not</em> show any genetic interaction if neither allele
has a phenotype in the tested condition.
To detect the genetic relationship between these alleles necessitates
identification of a condition that generates a phenotype for at least one
allele.
This limitation does not exist for synergistic alleles, which can account for
the fact that the number of negative genetic interactions far exceed the
number of positive genetic interactions in the genetic landscape of
<em>S. cerevisiae</em>
<span class="citation">(Costanzo et al. <a href="#ref-2016-00-Costanzo-Boone">2016</a>)</span>.
For our dE-MAP, we treated mutants with optimized concentrations of cisplatin
to enhance detection of both positive and negative genetic interactions.
The current network supports the notion that nucleotide excision repair (NER)
is the primary pathway responsible for <em>repairing</em> cisplatin lesions.
However, the lesion <em>bypassing</em> postreplication repair (PRR) pathway appears
to coordinate at least three critical nodes in the cisplatin tolerance
network, and is thus the most essential pathway for maintaining fitness in the
presence of cisplatin.
While our network was able to reveal these insights, it is clear that, even
with optimized treatment conditions, detecting epistasis remains challenging
using the dE-MAP approach alone.
An approach to overcome this limitation may be to use the conspicuous absence
of genetic interactions between otherwise highly correlated interaction
profiles to pick pairs of alleles for further fitness analysis using a
quantitative spot assay and a full cisplatin titration.</p>
</div>
<div id="tcga-biogrid-the-cell-map-insights-from-public-data" class="section level2">
<h2>TCGA, BioGRID &amp; The Cell Map: insights from public data</h2>
<p>Rad5, a member of PRR, was the focus of <strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a></strong>.
During a visit with Dr. Karlene Cimprich, she mentioned that there was
some evidence that the <em>RAD5</em> homolog, <em>HLTF</em>, was overexpressed
in cancer (also noted in the discussion by <span class="citation">Kile et al. (<a href="#ref-2015-06-Kile-Cimprich">2015</a>)</span>).
We therefore turned to publicly available sequencing data from The Cancer
Genome Atlas (TCGA) using software that enabled programmatic access to
gene-level summaries of this data
<span class="citation">(Cancer Genome Atlas Research Network <a href="#ref-2008-10-TCGA">2008</a>; Jacobsen and Questions <a href="#ref-cgdsr">2018</a>; Cerami et al. <a href="#ref-2012-05-Cerami-Schultz">2012</a>; Gao et al. <a href="#ref-2013-04-Gao-Schultz">2013</a>)</span>.
Indeed, the <em>RAD5</em> homolog, was frequently amplified and overexpressed in
several classes of squamous cell carcinomas.
This data-driven insight motivated the study of the genomic repercussions of
<em>RAD5</em> overexpression.
By screening the yeast gene disruption and temperature-sensitive mutant
collections we identified a clear signature of genetic requirements for <em>RAD5</em>
overexpression.
This signature led us to the discovery that, when overexpressed, Rad5 can
bypass canonical PRR regulatory signals to cause hyperactive recombination at
replication forks.
This hyperactive recombination is also accompanied by genome instability and
cisplatin sensitivity.</p>
<p>What was especially fascinating about the <em>RAD5</em> story was that both deletion
and overexpression caused cisplatin sensitivity, but only overexpression could
drive recombination.
This distinction was revealed by a careful comparison of the <em>RAD5</em>
overexpression and deletion genetic interaction profiles.
To compare interaction profiles, I used publicly available data from the
BioGRID interaction database <span class="citation">(Stark et al. <a href="#ref-2006-01-Stark">2006</a>)</span>, and I developed a landscape
enrichment analysis that uses easily accessible genetic interaction data from
The Cell Map
<span class="citation">(Usaj et al. <a href="#ref-2017-00-Usaj-Boone">2017</a>; Costanzo et al. <a href="#ref-2016-00-Costanzo-Boone">2016</a>; van der Maaten and Hinton <a href="#ref-2008-11-Maaten">2008</a>)</span>
(<strong><a href="https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg">Chapter 2, Figure 1</a> &amp;</strong>
<strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a>, Methods: Landscape enrichment analysis</strong>).
This landscape enrichment analysis proved to be an intuitive way to visualize
pathway enrichment for a genetic interaction profile, and allowed facile
exploration of the genetic landscape to address questions such as, “what
interactions are conspicuously absent?”
In the case of <em>rad5∆</em>, interactions with crossover resolution
(e.g. <em>top3</em> &amp; <em>sgs1∆</em>)
were clearly absent even though many of the mutant alleles in that region of
the landscape had been identified as having negative fitness interactions with
<em>rad5∆</em>
(<strong><a href="https://academic.oup.com/view-large/figure/163637138/gkz631fig2.jpg">Chapter 2, Figure 2</a></strong>).
Here, I would like to highlight that, in constructing our cisplatin dE-MAP, we
also observed no interaction between <em>rad5∆</em> and crossover resolution in the
untreated network, but, when we treated with cisplatin, we observed epistasis
– an example of conspicuous absence of synergy as weak evidence for epistasis
(<strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a>, Figure 4</strong>)!
Revealing the <em>rad5∆ sgs1∆</em> epistasis interaction in untreated conditions was
possible, but only when using a very sensitive low-throughput method
(<strong><a href="https://academic.oup.com/view-large/figure/163637138/gkz631fig2.jpg">Chapter 2, Figure 2</a></strong>).</p>
</div>
<div id="landscape-enrichment-embracing-perplexity" class="section level2">
<h2>Landscape enrichment: embracing perplexity</h2>
<p>As mentioned above, the landscape enrichment analysis from
<strong><a href="https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg">Chapter 2, Figure 1</a>C</strong> enabled facile exploration of publicly
available genetic interaction networks.
This analysis is similar to Systematic Functional Annotation and Visualization
of Biological Networks (SAFE), described by <span class="citation">Baryshnikova (<a href="#ref-2016-06-Baryshnikova">2016</a>)</span>, but differs
in its approach to dimension reduction
(i.e. the two-dimensional clustering approach used to generate the
“landscape”).
The implementation presented in the SAFE publication uses a spring-embedded
network layout, whereas the landscape presented for the <em>RAD5</em> chapter was
generated using t-SNE
<span class="citation">(van der Maaten and Hinton <a href="#ref-2008-11-Maaten">2008</a>)</span>.
The t-SNE method for dimension reduction is best summarized by Laurens van der
Maaten himself (<a href="https://lvdmaaten.github.io/tsne" class="uri">https://lvdmaaten.github.io/tsne</a>), which can take an
<span class="math inline">\(A \times N\)</span> matrix and reduce it to an <span class="math inline">\(A \times 2\)</span> matrix where the two
dimensions attempt to preserve local similarities between the profiles of <span class="math inline">\(A\)</span>.
For generating the genetic landscape of <em>Saccharomyces cerevisiae</em>, rows in
the matrix (<span class="math inline">\(A\)</span>) are genes, and columns in the matrix (<span class="math inline">\(N\)</span>) are genetic
profile correlation values.
Importantly, <span class="math inline">\(N\)</span> can be any number of features that might provide information
for generating clusters of genes or alleles.
Thus, the landscape enrichment approach can be easily adapted to integrate
other sources of data, and could be used to integrate existing biological
information to generate a genetic landscape for human genes.
t-SNE is lauded for its ability to reveal structure at many different scales.
An example of this can be seen in
<strong><a href="https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg">Chapter 2, Figure 1</a>C middle and right</strong>, where zooming in on the
dominant cluster, reveals additional local clustering of HDR, DNA replication,
and crossover resolution.
When viewing a t-SNE map it is important to be aware of the limitations of the
approach, and avoid some common misinterpretations, nicely described by
<span class="citation">Wattenberg, Vi’egas, and Johnson (<a href="#ref-2016-00-Wattenberg-Johnson">2016</a>)</span>.
Notably, relative distances between clusters on the landscape are largely
meaningless.
Clusters may appear on opposite sides of the landscape, but this does not mean
they are more dissimilar than a distinct cluster present half-way across the
landscape.
t-SNE also has a parameter, called “perplexity”, which loosely corresponds to
the expected number of members in a major cluster.
This parameter can have a major impact on clustering and must be lower than
the number of rows in the matrix (i.e. the size of <span class="math inline">\(A\)</span>, or the number of
genes).
Recommended values are between 5 and 50.
The default value of perplexity for the implementation of t-SNE in the R
programming language is 30 <span class="citation">(Krijthe <a href="#ref-Rtsne">2015</a>)</span>, and, unaware of its meaning, my first
attempt to run t-SNE on a matrix with only 29 rows resulted in the following
message, “<code>Error: Perplexity is too large</code>.” After a brief existential crisis,
I lowered the perplexity, which resulted in beautiful clustering similar to
that observed in
<strong><a href="https://doi.org/10.7916/d8-k1d0-kb09">Chapter 1</a>, Figure 6</strong>.
Paraphrasing a comment from Dr. Eugene Koonin on the quality of clusters,
“you know them when you see them.”</p>
</div>
<div id="quantifying-spots-replacing-the-drop-assay" class="section level2">
<h2>Quantifying spots: replacing the drop assay</h2>
<p>When working with high-throughput genetics, validating results is essential.
This often involves strain verification, back-crossing, and low-throughput
recapitulation of high-throughput observations using a gold-standard assay.
For yeast colony growth, owing mostly to simplicity and familiarity, the
serial dilution drop assay reigns supreme.
The serial dilution drop assay certainly works well for detecting obvious
fitness defects.
However, when combining multiple alleles to confirm potentially modest genetic
interactions, the qualitative nature of the drop assay quickly becomes a
burden, leading to more eye-squinting and head-scratching than insight.
For the <em>RAD5</em> manuscript we developed a quantitative colony growth assay
for generating high-resolution colony fitness estimates
(<strong><a href="https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg">Chapter 2, Figure 1</a>D &amp;</strong>
<strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">Chapter 2</a>, Methods: Genome-wide identification of SDL interactions</strong>).
In my experience, this assay is able to resolve fitness differences as small
as 5%.
A good example of this resolution can be seen in
<strong><a href="https://academic.oup.com/view-large/figure/163637138/gkz631fig2.jpg">Chapter 2, Figure 2</a>B</strong>.
While the effect is small, a significant positive interaction was detected
between <em>rad5∆</em> and <em>sgs1∆</em> in standard laboratory growth conditions –
additional evidence for epistasis between <em>rad5∆</em> and crossover resolution
without requiring cisplatin treatment!
The quantitative colony growth assay, while low in throughput, is
essential for validating our genetic observations from high-throughput
approaches like SGA and SPA.
With the CRISPR revolution beginning to produce combinatorial reverse genetic
screens, it will be greatly beneficial for the systems genetics community to
develop high-resolution validation approaches that are not subject to some of
the biases that can occur in these pooled screening approaches
<span class="citation">(Shen et al. <a href="#ref-2017-06-Shen-Mali">2017</a>; Sack et al. <a href="#ref-2016-09-Sack-Elledge">2016</a>)</span>.</p>
</div>
<div id="the-crispr-revolution-systems-genetics-in-new-model-systems" class="section level2">
<h2>The CRISPR revolution: systems genetics in new model systems</h2>
<p>Reverse genetic screens in mammalian cell lines have been possible through
high-throughput RNA interference experiments for nearly twenty years
<span class="citation">(Carpenter and Sabatini <a href="#ref-2004-01-Carpenter-Sabatini">2004</a>)</span>.
Work in this field has progressed methods for performing pooled screening
experiments to score fitness changes from a single transfected culture, wherein
loss or gain of library representation is detected through next-generation
sequencing
<span class="citation">(Sims et al. <a href="#ref-2011-10-Sims-Lord">2011</a>)</span>.
Initially, such experiments incurred high sequencing costs, and were plagued
by off-target effects and many irreproducible results
<span class="citation">(Bhinder and Djaballah <a href="#ref-2013-11-Bhinder-Djaballah">2013</a>)</span>.
While these limitations have largely been overcome through reduced sequencing
costs, formalized analysis pipelines, and improved experimental design
<span class="citation">(Schaefer et al. <a href="#ref-2018-00-Schaefer-Potratz">2018</a>)</span>,
the arrival of CRISPR appears to have taken functional genomics in mammalian
cells a major step forward
<span class="citation">(Evers et al. <a href="#ref-2016-06-Evers-Bernards">2016</a>)</span>.
The favored approach for CRISPR based screens currently appears to be using
Cas9 to induce double-strand breaks that, when improperly repaired, can result
in small insertions and deletions, potentially resulting in frameshift
mutations that may cause nonsense and gene disruption.
In <strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">Chapter 3</a></strong>, I presented a new CRISPR method, called iSTOP, that
allows direct generation of programmable nonsense without requiring a
double-strand break and mutagenic repair.
This feature may enable iSTOP to overcome the high levels of cell lethality
caused by Cas9-induced double-strand breaks
<span class="citation">(Haapaniemi et al. <a href="#ref-2018-07-Haapaniemi-Taipale">2018</a>; Ihry et al. <a href="#ref-2018-07-Ihry-Kaykas">2018</a>)</span>.
Importantly, for studying the DNA damage tolerance network in model systems
beyond yeast, iSTOP may avoid confounding effects caused by requiring repair
machinery to disrupt genes while simultaneously knocking out repair pathways.
For this reason, I look forward to analyzing results from experiments
comparing pooled screens using standard Cas9 and iSTOP.</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-2013-08-Alexandrov-Stratton">
<p>Alexandrov, Ludmil B, Serena Nik-Zainal, David C Wedge, Samuel A J R Aparicio, Sam Behjati, Andrew V Biankin, Graham R Bignell, et al. 2013. “Signatures of Mutational Processes in Human Cancer.” <em>Nature</em> 500 (7463): 415–21. <a href="https://doi.org/10.1038/nature12477">https://doi.org/10.1038/nature12477</a>.</p>
</div>
<div id="ref-2018-09-Ashworth-Lord">
<p>Ashworth, Alan, and Christopher J Lord. 2018. “Synthetic Lethal Therapies for Cancer: What’s Next After PARP Inhibitors?” <em>Nature Reviews. Clinical Oncology</em> 15 (9): 564–76. <a href="https://doi.org/10.1038/s41571-018-0055-6">https://doi.org/10.1038/s41571-018-0055-6</a>.</p>
</div>
<div id="ref-1944-02-Avery-McCarty">
<p>Avery, O T, C M Macleod, and M McCarty. 1944. “Studies on the Chemical Nature of the Substance Inducing Transformation of Pneumococcal Types: Induction of Transformation by a Desoxyribonucleic Acid Fraction Isolated from Pneumococcus Type III.” <em>The Journal of Experimental Medicine</em> 79 (2): 137–58. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19871359">https://www.ncbi.nlm.nih.gov/pubmed/19871359</a>.</p>
</div>
<div id="ref-2017-01-Baliga-Vonesch">
<p>Baliga, Nitin S, Johan L M Björkegren, Jef D Boeke, Michael Boutros, Nigel P S Crawford, Aim’ee M Dudley, Charles R Farber, et al. 2017. “The State of Systems Genetics in 2017.” <em>Cell Systems</em> 4 (1): 7–15. <a href="https://doi.org/10.1016/j.cels.2017.01.005">https://doi.org/10.1016/j.cels.2017.01.005</a>.</p>
</div>
<div id="ref-2010-12-Bandyopadhyay-Ideker">
<p>Bandyopadhyay, Sourav, Monika Mehta, Dwight Kuo, Min-Kyung Sung, Ryan Chuang, Eric J Jaehnig, Bernd Bodenmiller, et al. 2010. “Rewiring of Genetic Networks in Response to DNA Damage.” <em>Science (New York, N.Y.)</em> 330 (6009): 1385–9. <a href="https://doi.org/10.1126/science.1195618">https://doi.org/10.1126/science.1195618</a>.</p>
</div>
<div id="ref-2016-06-Baryshnikova">
<p>Baryshnikova, Anastasia. 2016. “Systematic Functional Annotation and Visualization of Biological Networks.” <em>Cell Systems</em> 2 (6): 412–21. <a href="https://doi.org/10.1016/j.cels.2016.04.014">https://doi.org/10.1016/j.cels.2016.04.014</a>.</p>
</div>
<div id="ref-2010-12-Baryshnikova-Myers">
<p>Baryshnikova, Anastasia, Michael Costanzo, Yungil Kim, Huiming Ding, Judice Koh, Kiana Toufighi, Ji-Young Youn, et al. 2010. “Quantitative Analysis of Fitness and Genetic Interactions in Yeast on a Genome Scale.” <em>Nature Methods</em> 7 (12): 1017–24. <a href="https://doi.org/10.1038/nmeth.1534">https://doi.org/10.1038/nmeth.1534</a>.</p>
</div>
<div id="ref-1993-07-Baudin-Cullin">
<p>Baudin, A, O Ozier-Kalogeropoulos, A Denouel, F Lacroute, and C Cullin. 1993. “A Simple and Efficient Method for Direct Gene Deletion in <em>Saccharomyces cerevisiae</em>.” <em>Nucleic Acids Research</em> 21 (14): 3329–30. <a href="https://www.ncbi.nlm.nih.gov/pubmed/8341614">https://www.ncbi.nlm.nih.gov/pubmed/8341614</a>.</p>
</div>
<div id="ref-2013-11-Bhinder-Djaballah">
<p>Bhinder, Bhavneet, and Hakim Djaballah. 2013. “Systematic Analysis of RNAi Reports Identifies Dismal Commonality at Gene-Level and Reveals an Unprecedented Enrichment in Pooled shRNA Screens.” <em>Combinatorial Chemistry &amp; High Throughput Screening</em> 16 (9): 665–81. <a href="https://www.ncbi.nlm.nih.gov/pubmed/23848309">https://www.ncbi.nlm.nih.gov/pubmed/23848309</a>.</p>
</div>
<div id="ref-2017-09-Billon-Bryant-Ciccia">
<p>Billon, Pierre, Eric E Bryant, Sarah A Joseph, Tarun S Nambiar, Samuel B Hayward, Rodney Rothstein, and Alberto Ciccia. 2017. “CRISPR-Mediated Base Editing Enables Efficient Disruption of Eukaryotic Genes Through Induction of STOP Codons.” <em>Molecular Cell</em> 67 (6): 1068–1079.e4. <a href="https://doi.org/10.1016/j.molcel.2017.08.008">https://doi.org/10.1016/j.molcel.2017.08.008</a>.</p>
</div>
<div id="ref-2002-06-Birrell-Brown">
<p>Birrell, Geoff W, James A Brown, H Irene Wu, Guri Giaever, Angela M Chu, Ronald W Davis, and J Martin Brown. 2002. “Transcriptional Response of <em>Saccharomyces cerevisiae</em> to DNA-Damaging Agents Does Not Identify the Genes That Protect Against These Agents.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 99 (13): 8778–83. <a href="https://doi.org/10.1073/pnas.132275199">https://doi.org/10.1073/pnas.132275199</a>.</p>
</div>
<div id="ref-2013-08-Braberg-Krogan">
<p>Braberg, Hannes, Huiyan Jin, Erica A Moehle, Yujia A Chan, Shuyi Wang, Michael Shales, Joris J Benschop, et al. 2013. “From Structure to Systems: High-Resolution, Quantitative Genetic Analysis of Rna Polymerase Ii.” <em>Cell</em> 154 (4): 775–88. <a href="https://doi.org/10.1016/j.cell.2013.07.033">https://doi.org/10.1016/j.cell.2013.07.033</a>.</p>
</div>
<div id="ref-1973-09-Brendel-Haynes">
<p>Brendel, M, and R H Haynes. 1973. “Interactions Among Genes Controlling Sensitivity to Radiation and Alkylation in Yeast.” <em>Molecular &amp; General Genetics: MGG</em> 125 (3): 197–216. <a href="https://www.ncbi.nlm.nih.gov/pubmed/4359562">https://www.ncbi.nlm.nih.gov/pubmed/4359562</a>.</p>
</div>
<div id="ref-2019-01-Bryant">
<p>Bryant, Eric E. 2019. “Systems Genetics of DNA Damage Tolerance – Cisplatin, RAD5 &amp;Amp; CRISPR-Mediated Nonsense.” <em>Columbia University</em>. <a href="https://doi.org/10.7916/d8-k1d0-kb09">https://doi.org/10.7916/d8-k1d0-kb09</a>.</p>
</div>
<div id="ref-2019-09-Bryant-Reid">
<p>Bryant, Eric E, Ivana Šunjevari’c, Luke Berchowitz, Rodney Rothstein, and Robert J D Reid. 2019. “Rad5 Dysregulation Drives Hyperactive Recombination at Replication Forks Resulting in Cisplatin Sensitivity and Genome Instability.” <em>Nucleic Acids Research</em> 47 (17): 9144–59. <a href="https://doi.org/10.1093/nar/gkz631">https://doi.org/10.1093/nar/gkz631</a>.</p>
</div>
<div id="ref-2008-10-TCGA">
<p>Cancer Genome Atlas Research Network. 2008. “Comprehensive Genomic Characterization Defines Human Glioblastoma Genes and Core Pathways.” <em>Nature</em> 455 (7216): 1061–8. <a href="https://doi.org/10.1038/nature07385">https://doi.org/10.1038/nature07385</a>.</p>
</div>
<div id="ref-2004-01-Carpenter-Sabatini">
<p>Carpenter, Anne E, and David M Sabatini. 2004. “Systematic Genome-Wide Screens of Gene Function.” <em>Nature Reviews. Genetics</em> 5 (1): 11–22. <a href="https://doi.org/10.1038/nrg1248">https://doi.org/10.1038/nrg1248</a>.</p>
</div>
<div id="ref-2012-05-Cerami-Schultz">
<p>Cerami, Ethan, Jianjiong Gao, Ugur Dogrusoz, Benjamin E Gross, Selcuk Onur Sumer, Bülent Arman Aksoy, Anders Jacobsen, et al. 2012. “The cBio Cancer Genomics Portal: An Open Platform for Exploring Multidimensional Cancer Genomics Data.” <em>Cancer Discovery</em> 2 (5): 401–4. <a href="https://doi.org/10.1158/2159-8290.CD-12-0095">https://doi.org/10.1158/2159-8290.CD-12-0095</a>.</p>
</div>
<div id="ref-2013-04-Cheng-Halmos">
<p>Cheng, Haiying, Zhenfeng Zhang, Alain Borczuk, Charles A Powell, Adayabalam S Balajee, Howard B Lieberman, and Balazs Halmos. 2013. “PARP Inhibition Selectively Increases Sensitivity to Cisplatin in ERCC1-Low Non-Small Cell Lung Cancer Cells.” <em>Carcinogenesis</em> 34 (4): 739–49. <a href="https://doi.org/10.1093/carcin/bgs393">https://doi.org/10.1093/carcin/bgs393</a>.</p>
</div>
<div id="ref-2010-10-Ciccia-Elledge">
<p>Ciccia, Alberto, and Stephen J Elledge. 2010. “The DNA Damage Response: Making It Safe to Play with Knives.” <em>Molecular Cell</em> 40 (2): 179–204. <a href="https://doi.org/10.1016/j.molcel.2010.09.019">https://doi.org/10.1016/j.molcel.2010.09.019</a>.</p>
</div>
<div id="ref-2016-01-Cleaver">
<p>Cleaver, James E. 2016. “Profile of Tomas Lindahl, Paul Modrich, and Aziz Sancar, 2015 Nobel Laureates in Chemistry.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 113 (2): 242–5. <a href="https://doi.org/10.1073/pnas.1521829112">https://doi.org/10.1073/pnas.1521829112</a>.</p>
</div>
<div id="ref-2007-04-Collins-Krogan">
<p>Collins, Sean R, Kyle M Miller, Nancy L Maas, Assen Roguev, Jeffrey Fillingham, Clement S Chu, Maya Schuldiner, et al. 2007. “Functional Dissection of Protein Complexes Involved in Yeast Chromosome Biology Using a Genetic Interaction Map.” <em>Nature</em> 446 (7137): 806–10. <a href="https://doi.org/10.1038/nature05649">https://doi.org/10.1038/nature05649</a>.</p>
</div>
<div id="ref-2006-00-Collins-Weissman">
<p>Collins, Sean R., Maya Schuldiner, Nevan J. Krogan, and Jonathan S. Weissman. 2006. “A Strategy for Extracting and Analyzing Large-Scale Quantitative Epistatic Interaction Data.” <em>Genome Biology</em> 7 (7): R63. <a href="https://doi.org/10.1186/gb-2006-7-7-r63">https://doi.org/10.1186/gb-2006-7-7-r63</a>.</p>
</div>
<div id="ref-2013-02-Cong-Zhang">
<p>Cong, Le, F Ann Ran, David Cox, Shuailiang Lin, Robert Barretto, Naomi Habib, Patrick D Hsu, et al. 2013. “Multiplex Genome Engineering Using CRISPR/Cas Systems.” <em>Science</em> 339 (6121): 819–23. <a href="https://doi.org/10.1126/science.1231143">https://doi.org/10.1126/science.1231143</a>.</p>
</div>
<div id="ref-2010-01-Costanzo-Boone">
<p>Costanzo, Michael, Anastasia Baryshnikova, Jeremy Bellay, Yungil Kim, Eric D Spear, Carolyn S Sevier, Huiming Ding, et al. 2010. “The Genetic Landscape of a Cell.” <em>Science</em> 327 (5964): 425–31. <a href="https://doi.org/10.1126/science.1180823">https://doi.org/10.1126/science.1180823</a>.</p>
</div>
<div id="ref-2016-00-Costanzo-Boone">
<p>Costanzo, Michael, Benjamin VanderSluis, Elizabeth N Koch, Anastasia Baryshnikova, Carles Pons, Guihong Tan, Wen Wang, et al. 2016. “A Global Genetic Interaction Network Maps a Wiring Diagram of Cellular Function.” <em>Science</em> 353 (6306). <a href="https://doi.org/10.1126/science.aaf1420">https://doi.org/10.1126/science.aaf1420</a>.</p>
</div>
<div id="ref-1968-00-Cox-Parry">
<p>Cox, B S, and J M Parry. 1968. “The Isolation, Genetics and Survival Characteristics of Ultraviolet Light-Sensitive Mutants in Yeast.” <em>Mutation Research</em> 6 (1): 37–55. <a href="https://www.ncbi.nlm.nih.gov/pubmed/5708072">https://www.ncbi.nlm.nih.gov/pubmed/5708072</a>.</p>
</div>
<div id="ref-1970-08-Crick">
<p>Crick, F. 1970. “Central Dogma of Molecular Biology.” <em>Nature</em> 227 (5258): 561–3. <a href="https://www.ncbi.nlm.nih.gov/pubmed/4913914">https://www.ncbi.nlm.nih.gov/pubmed/4913914</a>.</p>
</div>
<div id="ref-1961-12-Crick-WattsTobin">
<p>Crick, F H, L Barnett, S Brenner, and R J Watts-tobin. 1961. “General Nature of the Genetic Code for Proteins.” <em>Nature</em> 192 (December): 1227–32. <a href="https://www.ncbi.nlm.nih.gov/pubmed/13882203">https://www.ncbi.nlm.nih.gov/pubmed/13882203</a>.</p>
</div>
<div id="ref-2005-02-Dahm">
<p>Dahm, Ralf. 2005. “Friedrich Miescher and the Discovery of DNA.” <em>Developmental Biology</em> 278 (2): 274–88. <a href="https://doi.org/10.1016/j.ydbio.2004.11.028">https://doi.org/10.1016/j.ydbio.2004.11.028</a>.</p>
</div>
<div id="ref-2011-03-Deltcheva-Charpentier">
<p>Deltcheva, Elitza, Krzysztof Chylinski, Cynthia M Sharma, Karine Gonzales, Yanjie Chao, Zaid A Pirzada, Maria R Eckert, Jörg Vogel, and Emmanuelle Charpentier. 2011. “CRISPR RNA Maturation by Trans-Encoded Small RNA and Host Factor RNase III.” <em>Nature</em> 471 (7340): 602–7. <a href="https://doi.org/10.1038/nature09886">https://doi.org/10.1038/nature09886</a>.</p>
</div>
<div id="ref-2014-08-Dietlein-Reinhardt">
<p>Dietlein, Felix, Lisa Thelen, and H Christian Reinhardt. 2014. “Cancer-Specific Defects in DNA Repair Pathways as Targets for Personalized Therapeutic Approaches.” <em>Trends in Genetics</em> 30 (8): 326–39. <a href="https://doi.org/10.1016/j.tig.2014.06.003">https://doi.org/10.1016/j.tig.2014.06.003</a>.</p>
</div>
<div id="ref-2016-06-Evers-Bernards">
<p>Evers, Bastiaan, Katarzyna Jastrzebski, Jeroen P M Heijmans, Wipawadee Grernrum, Roderick L Beijersbergen, and Rene Bernards. 2016. “CRISPR Knockout Screening Outperforms shRNA and Crispri in Identifying Essential Genes.” <em>Nature Biotechnology</em> 34 (6): 631–3. <a href="https://doi.org/10.1038/nbt.3536">https://doi.org/10.1038/nbt.3536</a>.</p>
</div>
<div id="ref-2009-03-Fiedler-Krogan">
<p>Fiedler, Dorothea, Hannes Braberg, Monika Mehta, Gal Chechik, Gerard Cagney, Paromita Mukherjee, Andrea C Silva, et al. 2009. “Functional Organization of the <em>S. cerevisiae</em> Phosphorylation Network.” <em>Cell</em> 136 (5): 952–63. <a href="https://doi.org/10.1016/j.cell.2008.12.039">https://doi.org/10.1016/j.cell.2008.12.039</a>.</p>
</div>
<div id="ref-2005-00-Galanski-Keppler">
<p>Galanski, Markus, Michael A Jakupec, and Bernhard K Keppler. 2005. “Update of the Preclinical Situation of Anticancer Platinum Complexes: Novel Design Strategies and Innovative Analytical Approaches.” <em>Current Medicinal Chemistry</em> 12 (18): 2075–94. <a href="https://www.ncbi.nlm.nih.gov/pubmed/16101495">https://www.ncbi.nlm.nih.gov/pubmed/16101495</a>.</p>
</div>
<div id="ref-1974-09-Game-Mortimer">
<p>Game, J C, and R K Mortimer. 1974. “A Genetic Study of X-Ray Sensitive Mutants in Yeast.” <em>Mutation Research</em> 24 (3): 281–92. <a href="https://www.ncbi.nlm.nih.gov/pubmed/4606119">https://www.ncbi.nlm.nih.gov/pubmed/4606119</a>.</p>
</div>
<div id="ref-2013-04-Gao-Schultz">
<p>Gao, Jianjiong, Bülent Arman Aksoy, Ugur Dogrusoz, Gideon Dresdner, Benjamin Gross, S Onur Sumer, Yichao Sun, et al. 2013. “Integrative Analysis of Complex Cancer Genomics and Clinical Profiles Using the cBioPortal.” <em>Science Signaling</em> 6 (269): pl1. <a href="https://doi.org/10.1126/scisignal.2004088">https://doi.org/10.1126/scisignal.2004088</a>.</p>
</div>
<div id="ref-2010-11-Garneau-Moineau">
<p>Garneau, Josiane E, Marie-‘Eve Dupuis, Manuela Villion, Dennis A Romero, Rodolphe Barrangou, Patrick Boyaval, Christophe Fremaux, Philippe Horvath, Alfonso H Magad’an, and Sylvain Moineau. 2010. “The CRISPR/Cas Bacterial Immune System Cleaves Bacteriophage and Plasmid DNA.” <em>Nature</em> 468 (7320): 67–71. <a href="https://doi.org/10.1038/nature09523">https://doi.org/10.1038/nature09523</a>.</p>
</div>
<div id="ref-2012-09-Gasiunas-Siksnys">
<p>Gasiunas, Giedrius, Rodolphe Barrangou, Philippe Horvath, and Virginijus Siksnys. 2012. “Cas9-crRNA Ribonucleoprotein Complex Mediates Specific DNA Cleavage for Adaptive Immunity in Bacteria.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 109 (39): E2579–86. <a href="https://doi.org/10.1073/pnas.1208507109">https://doi.org/10.1073/pnas.1208507109</a>.</p>
</div>
<div id="ref-2002-07-Giaever-Johnston">
<p>Giaever, Guri, Angela M Chu, Li Ni, Carla Connelly, Linda Riles, Steeve V’eronneau, Sally Dow, et al. 2002. “Functional Profiling of the <em>Saccharomyces cerevisiae</em> Genome.” <em>Nature</em> 418 (6896): 387–91. <a href="https://doi.org/10.1038/nature00935">https://doi.org/10.1038/nature00935</a>.</p>
</div>
<div id="ref-1996-10-Goffeau-Oliver">
<p>Goffeau, A., B. G. Barrell, H. Bussey, R. W. Davis, B. Dujon, H. Feldmann, F. Galibert, et al. 1996. “Life with 6000 Genes.” <em>Science</em> 274 (5287): 546, 563–67.</p>
</div>
<div id="ref-1991-01-Goffeau-Vassarotti">
<p>Goffeau, A., and A. Vassarotti. 1991. “The European Project for Sequencing the Yeast Genome.” <em>Research in Microbiology</em> 142 (7): 901–3. <a href="https://doi.org/10.1016/0923-2508(91)90071-H">https://doi.org/10.1016/0923-2508(91)90071-H</a>.</p>
</div>
<div id="ref-2006-06-Grimm">
<p>Grimm, D. 2006. “A Mouse for Every Gene.” <em>Science</em> 312 (5782): 1862–6. <a href="https://doi.org/10.1126/science.312.5782.1862">https://doi.org/10.1126/science.312.5782.1862</a>.</p>
</div>
<div id="ref-2013-01-Gunol-vanAttikum">
<p>Gu’enol’e, Aude, Rohith Srivas, Kees Vreeken, Ze Zhong Wang, Shuyi Wang, Nevan J Krogan, Trey Ideker, and Haico van Attikum. 2013. “Dissection of DNA Damage Responses Using Multiconditional Genetic Interaction Maps.” <em>Molecular Cell</em> 49 (2): 346–58. <a href="https://doi.org/10.1016/j.molcel.2012.11.023">https://doi.org/10.1016/j.molcel.2012.11.023</a>.</p>
</div>
<div id="ref-2018-07-Haapaniemi-Taipale">
<p>Haapaniemi, Emma, Sandeep Botla, Jenna Persson, Bernhard Schmierer, and Jussi Taipale. 2018. “CRISPR-Cas9 Genome Editing Induces a p53-Mediated DNA Damage Response.” <em>Nature Medicine</em> 24 (7): 927–30. <a href="https://doi.org/10.1038/s41591-018-0049-z">https://doi.org/10.1038/s41591-018-0049-z</a>.</p>
</div>
<div id="ref-1997-11-Hartwell-Friend">
<p>Hartwell, L H, P Szankasi, C J Roberts, A W Murray, and S H Friend. 1997. “Integrating Genetic Approaches into the Discovery of Anticancer Drugs.” <em>Science</em> 278 (5340): 1064–8. <a href="https://www.ncbi.nlm.nih.gov/pubmed/9353181">https://www.ncbi.nlm.nih.gov/pubmed/9353181</a>.</p>
</div>
<div id="ref-2006-07-Hastie-Cohen">
<p>Hastie, C James, Cristina V’azquez-Martin, Amanda Philp, Michael J R Stark, and Patricia T W Cohen. 2006. “The <em>Saccharomyces cerevisiae</em> Orthologue of the Human Protein Phosphatase 4 Core Regulatory Subunit R2 Confers Resistance to the Anticancer Drug Cisplatin.” <em>The FEBS Journal</em> 273 (14): 3322–34. <a href="https://doi.org/10.1111/j.1742-4658.2006.05336.x">https://doi.org/10.1111/j.1742-4658.2006.05336.x</a>.</p>
</div>
<div id="ref-1981-00-Haynes-Kunz">
<p>Haynes, Robert, and Bernard Kunz. 1981. “DNA Repair and Mutagenesis in Yeast.” <em>Cold Spring Harbor Monograph Archive</em> 11A (0). <a href="https://cshmonographs.org/index.php/monographs/article/view/4235/3411">https://cshmonographs.org/index.php/monographs/article/view/4235/3411</a>.</p>
</div>
<div id="ref-2010-00-Heyer">
<p>Heyer, Wolf-Dietrich, Kirk T Ehmsen, and Jie Liu. 2010. “Regulation of Homologous Recombination in Eukaryotes.” <em>Annual Review of Genetics</em> 44: 113–39. <a href="https://doi.org/10.1146/annurev-genet-051710-150955">https://doi.org/10.1146/annurev-genet-051710-150955</a>.</p>
</div>
<div id="ref-2008-04-Hillenmeyer-Giaever">
<p>Hillenmeyer, Maureen E, Eula Fung, Jan Wildenhain, Sarah E Pierce, Shawn Hoon, William Lee, Michael Proctor, et al. 2008. “The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes.” <em>Science</em> 320 (5874): 362–5. <a href="https://doi.org/10.1126/science.1150021">https://doi.org/10.1126/science.1150021</a>.</p>
</div>
<div id="ref-2005-07-Huang-Kowalski">
<p>Huang, Ruea-Yea, Martha Eddy, Marija Vujcic, and David Kowalski. 2005. “Genome-Wide Screen Identifies Genes Whose Inactivation Confer Resistance to Cisplatin in <em>Saccharomyces cerevisiae</em>.” <em>Cancer Research</em> 65 (13): 5890–7. <a href="https://doi.org/10.1158/0008-5472.CAN-04-4093">https://doi.org/10.1158/0008-5472.CAN-04-4093</a>.</p>
</div>
<div id="ref-2015-01-Hustedt-Gasser">
<p>Hustedt, Nicole, Andrew Seeber, Ragna Sack, Monika Tsai-Pflugfelder, Bhupinder Bhullar, Hanneke Vlaming, Fred van Leeuwen, et al. 2015. “Yeast PP4 Interacts with ATR Homolog Ddc2-Mec1 and Regulates Checkpoint Signaling.” <em>Molecular Cell</em> 57 (2): 273–89. <a href="https://doi.org/10.1016/j.molcel.2014.11.016">https://doi.org/10.1016/j.molcel.2014.11.016</a>.</p>
</div>
<div id="ref-2012-01-Ideker-Krogan">
<p>Ideker, Trey, and Nevan J Krogan. 2012. “Differential Network Biology.” <em>Molecular Systems Biology</em> 8 (January): 565. <a href="https://doi.org/10.1038/msb.2011.99">https://doi.org/10.1038/msb.2011.99</a>.</p>
</div>
<div id="ref-2018-07-Ihry-Kaykas">
<p>Ihry, Robert J, Kathleen A Worringer, Max R Salick, Elizabeth Frias, Daniel Ho, Kraig Theriault, Sravya Kommineni, et al. 2018. “p53 Inhibits CRISPR-Cas9 Engineering in Human Pluripotent Stem Cells.” <em>Nature Medicine</em> 24 (7): 939–46. <a href="https://doi.org/10.1038/s41591-018-0050-6">https://doi.org/10.1038/s41591-018-0050-6</a>.</p>
</div>
<div id="ref-cgdsr">
<p>Jacobsen, Anders, and cBioPortal Questions. 2018. <em>cgdsr: R-Based API for Accessing the MSKCC Cancer Genomics Data Server (CGDS)</em>. <a href="https://CRAN.R-project.org/package=cgdsr">https://CRAN.R-project.org/package=cgdsr</a>.</p>
</div>
<div id="ref-2012-08-Jinek-Charpentier">
<p>Jinek, Martin, Krzysztof Chylinski, Ines Fonfara, Michael Hauer, Jennifer A Doudna, and Emmanuelle Charpentier. 2012. “A Programmable Dual-RNA-Guided DNA Endonuclease in Adaptive Bacterial Immunity.” <em>Science</em> 337 (6096): 816–21. <a href="https://doi.org/10.1126/science.1225829">https://doi.org/10.1126/science.1225829</a>.</p>
</div>
<div id="ref-2007-08-Kelland">
<p>Kelland, Lloyd. 2007. “The Resurgence of Platinum-Based Cancer Chemotherapy.” <em>Nature Reviews. Cancer</em> 7 (8): 573–84. <a href="https://doi.org/10.1038/nrc2167">https://doi.org/10.1038/nrc2167</a>.</p>
</div>
<div id="ref-2015-06-Kile-Cimprich">
<p>Kile, Andrew C, Diana A Chavez, Julien Bacal, Sherif Eldirany, Dmitry M Korzhnev, Irina Bezsonova, Brandt F Eichman, and Karlene A Cimprich. 2015. “HLTF’s Ancient HIRAN Domain Binds 3’ DNA Ends to Drive Replication Fork Reversal.” <em>Molecular Cell</em> 58 (6): 1090–1100. <a href="https://doi.org/10.1016/j.molcel.2015.05.013">https://doi.org/10.1016/j.molcel.2015.05.013</a>.</p>
</div>
<div id="ref-2012-08-Knock-Berneburg">
<p>Knoch, Jennifer, York Kamenisch, Christian Kubisch, and Mark Berneburg. 2012. “Rare Hereditary Diseases with Defects in DNA-Repair.” <em>European Journal of Dermatology</em> 22 (4): 443–55. <a href="https://doi.org/10.1684/ejd.2012.1654">https://doi.org/10.1684/ejd.2012.1654</a>.</p>
</div>
<div id="ref-1990-06-Koller-Smithies">
<p>Koller, B H, P Marrack, J W Kappler, and O Smithies. 1990. “Normal Development of Mice Deficient in Beta 2M, MHC Class I Proteins, and CD8+ T Cells.” <em>Science</em> 248 (4960): 1227–30. <a href="https://www.ncbi.nlm.nih.gov/pubmed/2112266">https://www.ncbi.nlm.nih.gov/pubmed/2112266</a>.</p>
</div>
<div id="ref-1989-11-Koller-Smithies">
<p>Koller, B H, and O Smithies. 1989. “Inactivating the Beta 2-Microglobulin Locus in Mouse Embryonic Stem Cells by Homologous Recombination.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 86 (22): 8932–5. <a href="https://www.ncbi.nlm.nih.gov/pubmed/2682666">https://www.ncbi.nlm.nih.gov/pubmed/2682666</a>.</p>
</div>
<div id="ref-2016-05-Komor-Liu">
<p>Komor, Alexis C, Yongjoo B Kim, Michael S Packer, John A Zuris, and David R Liu. 2016. “Programmable Editing of a Target Base in Genomic DNA Without Double-Stranded DNA Cleavage.” <em>Nature</em> 533 (7603): 420–4. <a href="https://doi.org/10.1038/nature17946">https://doi.org/10.1038/nature17946</a>.</p>
</div>
<div id="ref-Rtsne">
<p>Krijthe, Jesse H. 2015. <em>Rtsne: T-Distributed Stochastic Neighbor Embedding Using Barnes-Hut Implementation</em>. <a href="https://github.com/jkrijthe/Rtsne">https://github.com/jkrijthe/Rtsne</a>.</p>
</div>
<div id="ref-2001-02-Lander-IHGSC">
<p>Lander, E S, L M Linton, B Birren, C Nusbaum, M C Zody, J Baldwin, K Devon, et al. 2001. “Initial Sequencing and Analysis of the Human Genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div id="ref-2005-08-Lee-Giaever">
<p>Lee, William, Robert P St Onge, Michael Proctor, Patrick Flaherty, Michael I Jordan, Adam P Arkin, Ronald W Davis, Corey Nislow, and Guri Giaever. 2005. “Genome-Wide Requirements for Resistance to Functionally Distinct DNA-Damaging Agents.” <em>PLoS Genetics</em> 1 (2): e24. <a href="https://doi.org/10.1371/journal.pgen.0010024">https://doi.org/10.1371/journal.pgen.0010024</a>.</p>
</div>
<div id="ref-1971-05-Lemontt">
<p>Lemontt, J F. 1971. “Mutants of Yeast Defective in Mutation Induced by Ultraviolet Light.” <em>Genetics</em> 68 (1): 21–33. <a href="https://www.ncbi.nlm.nih.gov/pubmed/17248528">https://www.ncbi.nlm.nih.gov/pubmed/17248528</a>.</p>
</div>
<div id="ref-2007-02-Liao-Panaretou">
<p>Liao, Chunyan, Bin Hu, Matthew J Arno, and Barry Panaretou. 2007. “Genomic Screening <em>in vivo</em> Reveals the Role Played by Vacuolar H+ ATPase and Cytosolic Acidification in Sensitivity to DNA-Damaging Agents Such as Cisplatin.” <em>Molecular Pharmacology</em> 71 (2): 416–25. <a href="https://doi.org/10.1124/mol.106.030494">https://doi.org/10.1124/mol.106.030494</a>.</p>
</div>
<div id="ref-1993-04-Lindahl">
<p>Lindahl, T. 1993. “Instability and Decay of the Primary Structure of DNA.” <em>Nature</em> 362 (6422): 709–15. <a href="https://doi.org/10.1038/362709a0">https://doi.org/10.1038/362709a0</a>.</p>
</div>
<div id="ref-2015-00-Lord-Ashworth">
<p>Lord, Christopher J, Andrew N J Tutt, and Alan Ashworth. 2015. “Synthetic Lethality and Cancer Therapy: Lessons Learned from the Development of PARP Inhibitors.” <em>Annual Review of Medicine</em> 66: 455–70. <a href="https://doi.org/10.1146/annurev-med-050913-022545">https://doi.org/10.1146/annurev-med-050913-022545</a>.</p>
</div>
<div id="ref-2013-02-Mali-Church">
<p>Mali, Prashant, Luhan Yang, Kevin M Esvelt, John Aach, Marc Guell, James E DiCarlo, Julie E Norville, and George M Church. 2013. “RNA-Guided Human Genome Engineering via Cas9.” <em>Science</em> 339 (6121): 823–6. <a href="https://doi.org/10.1126/science.1232033">https://doi.org/10.1126/science.1232033</a>.</p>
</div>
<div id="ref-2017-11-Nickoloff-Hromas">
<p>Nickoloff, Jac A, Dennie Jones, Suk-Hee Lee, Elizabeth A Williamson, and Robert Hromas. 2017. “Drugging the Cancers Addicted to DNA Repair.” <em>Journal of the National Cancer Institute</em> 109 (11). <a href="https://doi.org/10.1093/jnci/djx059">https://doi.org/10.1093/jnci/djx059</a>.</p>
</div>
<div id="ref-1966-00-Nirenberg-Anderson">
<p>Nirenberg, M., T. Caskey, R. Marshall, R. Brimacombe, D. Kellogg, B. Doctor, D. Hatfield, et al. 1966. “The RNA Code and Protein Synthesis.” <em>Cold Spring Harbor Symposia on Quantitative Biology</em> 31: 11–24.</p>
</div>
<div id="ref-2015-11-Connor">
<p>O’Connor, Mark J. 2015. “Targeting the DNA Damage Response in Cancer.” <em>Molecular Cell</em> 60 (4): 547–60. <a href="https://doi.org/10.1016/j.molcel.2015.10.040">https://doi.org/10.1016/j.molcel.2015.10.040</a>.</p>
</div>
<div id="ref-2006-03-Pan-Boeke">
<p>Pan, Xuewen, Ping Ye, Daniel S Yuan, Xiaoling Wang, Joel S Bader, and Jef D Boeke. 2006. “A DNA Integrity Network in the Yeast <em>Saccharomyces cerevisiae</em>.” <em>Cell</em> 124 (5): 1069–81. <a href="https://doi.org/10.1016/j.cell.2005.12.036">https://doi.org/10.1016/j.cell.2005.12.036</a>.</p>
</div>
<div id="ref-2018-09-Rocha">
<p>Rocha, Clarissa Ribeiro Reily, Matheus Molina Silva, Annabel Quinet, Januario Bispo Cabral-Neto, and Carlos Frederico Martins Menck. 2018. “DNA Repair Pathways and Cisplatin Resistance: An Intimate Relationship.” <em>Clinics</em> 73 (suppl 1): e478s. <a href="https://doi.org/10.6061/clinics/2018/e478s">https://doi.org/10.6061/clinics/2018/e478s</a>.</p>
</div>
<div id="ref-1983-01-Rothstein">
<p>Rothstein, Rodney J. 1983. “One-Step Gene Disruption in Yeast.” <em>Methods in Enzymology</em> 101 (101). Elsevier: 202–11. <a href="https://doi.org/10.1016/0076-6879(83)01015-0">https://doi.org/10.1016/0076-6879(83)01015-0</a>.</p>
</div>
<div id="ref-2011-12-Roy-Powell">
<p>Roy, Rohini, Jarin Chun, and Simon N Powell. 2011. “BRCA1 and BRCA2: Different Roles in a Common Pathway of Genome Protection.” <em>Nature Reviews. Cancer</em> 12 (1): 68–78. <a href="https://doi.org/10.1038/nrc3181">https://doi.org/10.1038/nrc3181</a>.</p>
</div>
<div id="ref-2016-09-Sack-Elledge">
<p>Sack, Laura Magill, Teresa Davoli, Qikai Xu, Mamie Z Li, and Stephen J Elledge. 2016. “Sources of Error in Mammalian Genetic Screens.” <em>G3</em> 6 (9): 2781–90. <a href="https://doi.org/10.1534/g3.116.030973">https://doi.org/10.1534/g3.116.030973</a>.</p>
</div>
<div id="ref-2011-11-Sapranauskas-Siksnys">
<p>Sapranauskas, Rimantas, Giedrius Gasiunas, Christophe Fremaux, Rodolphe Barrangou, Philippe Horvath, and Virginijus Siksnys. 2011. “The <em>Streptococcus thermophilus</em> CRISPR/Cas System Provides Immunity in <em>Escherichia coli</em>.” <em>Nucleic Acids Research</em> 39 (21): 9275–82. <a href="https://doi.org/10.1093/nar/gkr606">https://doi.org/10.1093/nar/gkr606</a>.</p>
</div>
<div id="ref-2018-00-Schaefer-Potratz">
<p>Schaefer, Christiane, Nikhil Mallela, Jochen Seggewiß, Birgit Lechtape, Heymut Omran, Uta Dirksen, Eberhard Korsching, and Jenny Potratz. 2018. “Target Discovery Screens Using Pooled shRNA Libraries and Next-Generation Sequencing: A Model Workflow and Analytical Algorithm.” <em>PloS One</em> 13 (1): e0191570. <a href="https://doi.org/10.1371/journal.pone.0191570">https://doi.org/10.1371/journal.pone.0191570</a>.</p>
</div>
<div id="ref-2005-11-Schuldiner-Krogan">
<p>Schuldiner, Maya, Sean R Collins, Natalie J Thompson, Vladimir Denic, Arunashree Bhamidipati, Thanuja Punna, Jan Ihmels, et al. 2005. “Exploration of the Function and Organization of the Yeast Early Secretory Pathway Through an Epistatic Miniarray Profile.” <em>Cell</em> 123 (3): 507–19. <a href="https://doi.org/10.1016/j.cell.2005.08.031">https://doi.org/10.1016/j.cell.2005.08.031</a>.</p>
</div>
<div id="ref-2006-12-Schuldiner-Krogan">
<p>Schuldiner, M, S R Collins, J S Weissman, and N J Krogan. 2006. “Quantitative Genetic Analysis in <em>Saccharomyces cerevisiae</em> Using Epistatic Miniarray Profiles (E-MAPs) and Its Application to Chromatin Functions.” <em>Methods</em> 40 (4): 344–52. <a href="https://doi.org/10.1016/j.ymeth.2006.07.034">https://doi.org/10.1016/j.ymeth.2006.07.034</a>.</p>
</div>
<div id="ref-2017-06-Shen-Mali">
<p>Shen, John Paul, Dongxin Zhao, Roman Sasik, Jens Luebeck, Amanda Birmingham, Ana Bojorquez-Gomez, Katherine Licon, et al. 2017. “Combinatorial CRISPR-Cas9 Screens for <em>de novo</em> Mapping of Genetic Interactions.” <em>Nature Methods</em> 14 (6): 573–76. <a href="https://doi.org/10.1038/nmeth.4225">https://doi.org/10.1038/nmeth.4225</a>.</p>
</div>
<div id="ref-2000-01-Simon-Friend">
<p>Simon, J A, P Szankasi, D K Nguyen, C Ludlow, H M Dunstan, C J Roberts, E L Jensen, L H Hartwell, and S H Friend. 2000. “Differential Toxicities of Anticancer Agents Among DNA Repair and Checkpoint Mutants of <em>Saccharomyces cerevisiae</em>.” <em>Cancer Research</em> 60 (2): 328–33. <a href="https://www.ncbi.nlm.nih.gov/pubmed/10667584">https://www.ncbi.nlm.nih.gov/pubmed/10667584</a>.</p>
</div>
<div id="ref-2011-10-Sims-Lord">
<p>Sims, David, Ana M Mendes-Pereira, Jessica Frankum, Darren Burgess, Maria-Antonietta Cerone, Cristina Lombardelli, Costas Mitsopoulos, et al. 2011. “High-Throughput RNA Interference Screening Using Pooled shRNA Libraries and Next Generation Sequencing.” <em>Genome Biology</em> 12 (10): R104. <a href="https://doi.org/10.1186/gb-2011-12-10-r104">https://doi.org/10.1186/gb-2011-12-10-r104</a>.</p>
</div>
<div id="ref-1967-09-Snow">
<p>Snow, R. 1967. “Mutants of Yeast Sensitive to Ultraviolet Light.” <em>Journal of Bacteriology</em> 94 (3): 571–5. <a href="https://www.ncbi.nlm.nih.gov/pubmed/6035260">https://www.ncbi.nlm.nih.gov/pubmed/6035260</a>.</p>
</div>
<div id="ref-2013-12-Srivas-Ideker">
<p>Srivas, Rohith, Thomas Costelloe, Anne-Ruxandra Carvunis, Sovan Sarkar, Erik Malta, Su Ming Sun, Marijke Pool, et al. 2013. “A UV-Induced Genetic Network Links the RSC Complex to Nucleotide Excision Repair and Shows Dose-Dependent Rewiring.” <em>Cell Reports</em> 5 (6): 1714–24. <a href="https://doi.org/10.1016/j.celrep.2013.11.035">https://doi.org/10.1016/j.celrep.2013.11.035</a>.</p>
</div>
<div id="ref-2006-01-Stark">
<p>Stark, Chris, Bobby-Joe Breitkreutz, Teresa Reguly, Lorrie Boucher, Ashton Breitkreutz, and Mike Tyers. 2006. “BioGRID: A General Repository for Interaction Datasets.” <em>Nucleic Acids Research</em> 34 (Database issue): D535–9. <a href="https://doi.org/10.1093/nar/gkj109">https://doi.org/10.1093/nar/gkj109</a>.</p>
</div>
<div id="ref-2007-02-Onge-Giaever">
<p>St. Onge, Robert P, Ramamurthy Mani, Julia Oh, Michael Proctor, Eula Fung, Ronald W Davis, Corey Nislow, Frederick P Roth, and Guri Giaever. 2007. “Systematic Pathway Analysis Using High-Resolution Fitness Profiling of Combinatorial Gene Deletions.” <em>Nature Genetics</em> 39 (2): 199–206. <a href="https://doi.org/10.1038/ng1948">https://doi.org/10.1038/ng1948</a>.</p>
</div>
<div id="ref-2001-12-Tong-Boone">
<p>Tong, A H, M Evangelista, A B Parsons, H Xu, G D Bader, N Pa ’e, M Robinson, et al. 2001. “Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants.” <em>Science</em> 294 (5550): 2364–8. <a href="https://doi.org/10.1126/science.1065810">https://doi.org/10.1126/science.1065810</a>.</p>
</div>
<div id="ref-2006-00-Tong-Boone">
<p>Tong, Amy Hin Yan, and Charles Boone. 2006. “Synthetic Genetic Array Analysis in <em>Saccharomyces Cerevisiae</em>.” <em>Methods in Molecular Biology</em> 313: 171–92. <a href="https://www.ncbi.nlm.nih.gov/pubmed/16118434">https://www.ncbi.nlm.nih.gov/pubmed/16118434</a>.</p>
</div>
<div id="ref-2017-00-Usaj-Boone">
<p>Usaj, Matej, Yizhao Tan, Wen Wang, Benjamin VanderSluis, Albert Zou, Chad L Myers, Michael Costanzo, Brenda Andrews, and Charles Boone. 2017. “TheCellMap.Org: A Web-Accessible Database for Visualizing and Mining the Global Yeast Genetic Interaction Network.” <em>G3</em> 7 (5): 1539–49. <a href="https://doi.org/10.1534/g3.117.040220">https://doi.org/10.1534/g3.117.040220</a>.</p>
</div>
<div id="ref-2008-11-Maaten">
<p>van der Maaten, Laurens, and Geoffrey Hinton. 2008. “Visualizing Data Using t-SNE.” <em>Journal of Machine Learning Research</em> 9 (Nov): 2579–2605. <a href="http://www.jmlr.org/papers/v9/vandermaaten08a.html">http://www.jmlr.org/papers/v9/vandermaaten08a.html</a>.</p>
</div>
<div id="ref-2007-10-Vogel">
<p>Vogel, Gretchen. 2007. “Nobel Prizes. A Knockout Award in Medicine.” <em>Science</em> 318 (5848): 178–9. <a href="https://doi.org/10.1126/science.318.5848.178">https://doi.org/10.1126/science.318.5848.178</a>.</p>
</div>
<div id="ref-1994-12-Wach-Philippsen">
<p>Wach, A, A Brachat, R Pöhlmann, and P Philippsen. 1994. “New Heterologous Modules for Classical or PCR-Based Gene Disruptions in <em>Saccharomyces cerevisiae</em>.” <em>Yeast</em> 10 (13): 1793–1808. <a href="https://www.ncbi.nlm.nih.gov/pubmed/7747518">https://www.ncbi.nlm.nih.gov/pubmed/7747518</a>.</p>
</div>
<div id="ref-2002-12-Waterston">
<p>Waterston, Robert H, Kerstin Lindblad-Toh, Ewan Birney, Jane Rogers, Josep F Abril, Pankaj Agarwal, Richa Agarwala, et al. 2002. “Initial Sequencing and Comparative Analysis of the Mouse Genome.” <em>Nature</em> 420 (6915): 520–62. <a href="https://doi.org/10.1038/nature01262">https://doi.org/10.1038/nature01262</a>.</p>
</div>
<div id="ref-1953-04-Watson-Crick">
<p>Watson, J D, and F H Crick. 1953. “Molecular Structure of Nucleic Acids; a Structure for Deoxyribose Nucleic Acid.” <em>Nature</em> 171 (4356): 737–8. <a href="https://www.ncbi.nlm.nih.gov/pubmed/13054692">https://www.ncbi.nlm.nih.gov/pubmed/13054692</a>.</p>
</div>
<div id="ref-2016-00-Wattenberg-Johnson">
<p>Wattenberg, Martin, Fernanda Vi’egas, and Ian Johnson. 2016. “How to Use T-Sne Effectively.” <em>Distill</em>. <a href="https://doi.org/10.23915/distill.00002">https://doi.org/10.23915/distill.00002</a>.</p>
</div>
<div id="ref-1999-08-Winzeler-Davis">
<p>Winzeler, E A, D D Shoemaker, A Astromoff, H Liang, K Anderson, B Andre, R Bangham, et al. 1999. “Functional Characterization of the <em>S. cerevisiae</em> Genome by Gene Deletion and Parallel Analysis.” <em>Science</em> 285 (5429): 901–6. <a href="https://www.ncbi.nlm.nih.gov/pubmed/10436161">https://www.ncbi.nlm.nih.gov/pubmed/10436161</a>.</p>
</div>
<div id="ref-2004-06-Wu-Brown">
<p>Wu, H Irene, James A Brown, Mary J Dorie, Laura Lazzeroni, and J Martin Brown. 2004. “Genome-Wide Identification of Genes Conferring Resistance to the Anticancer Agents Cisplatin, Oxaliplatin, and Mitomycin C.” <em>Cancer Research</em> 64 (11): 3940–8. <a href="https://doi.org/10.1158/0008-5472.CAN-03-3113">https://doi.org/10.1158/0008-5472.CAN-03-3113</a>.</p>
</div>
<div id="ref-2013-02-Zhang">
<p>Zhang, Junran. 2013. “The Role of BRCA1 in Homologous Recombination Repair in Response to Replication Stress: Significance in Tumorigenesis and Cancer Therapy.” <em>Cell &amp; Bioscience</em> 3 (1): 11. <a href="https://doi.org/10.1186/2045-3701-3-11">https://doi.org/10.1186/2045-3701-3-11</a>.</p>
</div>
<div id="ref-1990-04-Zijlstra-Jaenisch">
<p>Zijlstra, M, M Bix, N E Simister, J M Loring, D H Raulet, and R Jaenisch. 1990. “Beta 2-Microglobulin Deficient Mice Lack CD4-8+ Cytolytic T Cells.” <em>Nature</em> 344 (6268): 742–6. <a href="https://doi.org/10.1038/344742a0">https://doi.org/10.1038/344742a0</a>.</p>
</div>
<div id="ref-1989-11-Zijlstra-Jaenisch">
<p>Zijlstra, M, E Li, F Sajjadi, S Subramani, and R Jaenisch. 1989. “Germ-Line Transmission of a Disrupted Beta 2-Microglobulin Gene Produced by Homologous Recombination in Embryonic Stem Cells.” <em>Nature</em> 342 (6248): 435–8. <a href="https://doi.org/10.1038/342435a0">https://doi.org/10.1038/342435a0</a>.</p>
</div>
</div>
</div>
